src/devyatovskaya/Task3/PreparedScenes.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include "GLScene.h"
+#include <vector>
+
+class PreparedScenes {
+public:
+  std::vector<GLScene> scenes;
+  void init();
+
+private:
+  GLScene default_morphing(ShaderData &data);
```

Comment:
Если эти методы не модифицируют ShaderData, то лучше сделать метод константным и данные передавать по const ссылке
```suggestion
  [[nodiscard]] GLScene default_morphing(const ShaderData &data);
```